metricq_wizard_backend/metricq/configurator.py

```diff
@@ -82,6 +93,25 @@ async def connect(self):
         # After that, we do the MetricQ connection stuff
         await super().connect()
 
+    @cached(ttl=5 * 60, cache=SimpleMemoryCache)
+    async def fetch_bindings(self):
+        bindings = defaultdict(list)
+        async with aiohttp.ClientSession() as session:
+            async with session.get(
+                self.rabbitmq_url + "/api/exchanges/data/metricq.data/bindings/source",
+                auth=aiohttp.BasicAuth(self.rabbitmq_user, self.rabbitmq_password),
+            ) as resp:
+                for binding in await resp.json():
+                    metric = binding["routing_key"]
+                    consumer = binding["destination"][:-5]
```

Comment:
`-5`, because `len("-data") == 5`?

Reply:
```suggestion
                    consumer = binding["destination"].removesuffix("-data")
```
From your own suggestions :smile: